pages/project-version/pdf/views/header.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
-import Wrapper from './wrapper';
 import moment from 'moment';
+import Wrapper from './wrapper';
+import StatusBanner from './status-banner';
 
 const format = date => moment(date).format('DD MMM YY');
 
-const Header = ({ project }) => (
-  <Wrapper name="header">
+const Header = ({ project, nonce }) => (
+  <Wrapper name="header" nonce={nonce}>
     <header>
+      <StatusBanner licence={project} licenceType="ppl" />
```

Comment:
Do we _always_ want a status banner? If so is the previous conditional with different padding now redundant?

Reply:
OIC, so the StatusBanner returns `null` if active - as you were :)